### ISSUE TITLE
Way to show horizontal grid lines but not vertical ones

### ISSUE THREAD

**Issue by Chrriis** [JFormDesigner/FlatLaf#38]
It is quite hard to follow the values of a given item (row) when tables have no lines at all.

Generally, tables in Flat design have, or allow activation of, horizontal lines (but not necessarily vertical lines). I did not find any way to do it with FlatLaf, so sadly I have to show all lines using:
```Java
UIDefaults defaults = UIManager.getDefaults();
defaults.put("Table.showGrid", Boolean.TRUE);
defaults.put("Table.intercellSpacing", 1);
```

Am I missing something? Is there a way to activate horizontal lines only, and that as a global L&F setting (not per table instance like the `setShowHorizontalLines(b)` method)?

**Comment by j-dimension** [comment]
I am interested in that as well.

**Comment by DevCharly** [comment]
Ok, good idea.

I'll replace `Table.showGrid` with `Table.showHorizontalLines` and `Table.showVerticalLines`

**Comment by DevCharly** [comment]
The change is now in master branch

Use following to enable horizontal lines for all tables:
~~~java
UIManager.put( "Table.showHorizontalLines", true );
~~~

If you want to see horizontal lines when multiple rows are selected, then use following (but I think that this is not needed):
~~~java
UIManager.put( "Table.intercellSpacing", new Dimension( 0, 1 ) );
~~~

**Comment by Chrriis** [comment]
I just tested and it works!

Now I have an odd behavior in my code when I change row backgrounds, because these rows show vertical lines between cells... But as I said, it is in my code 😉 

Thanks a lot!

**Comment by DevCharly** [comment]
fixed in 0.24
